ext/riverlea/streams/thames/css/civicrm.css

```diff
@@ -382,6 +382,14 @@ div:has(> table) {
   overflow-x: auto;
   max-width: 100%;
 }
+/* When a menu is activated, allow it 20rem height to prevent it causing scroll
+ * when the menu is opened on a row near the bottom of the table and
+ * which the user might not notice / might make the menu hard to access.
+ * https://lab.civicrm.org/extensions/riverlea/-/issues/90
+ */
+div:has(> table .btn-slide-active, > table .dropdown-toggle[aria-expanded="true"]) {
```

Comment:
@vingle 
I think the linter would prefer
```
div:has(> table .btn-slide-active),
div:has(> table .dropdown-toggle[aria-expanded="true"])
```

Bit more verbose but computers need help reading :)

Reply:
Me too! Thanks @ufundo. Changed (cc @artfulrobot).